tests/PySys/apt_plugin/apt_sequence_remove_with_version_fail/run.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import os
+import time
+
+sys.path.append("apt_plugin")
+from environment_apt_plugin import AptPlugin
+
+"""
+Validate apt plugin remove a specific versioned module use case
+
+When we remove a package module with version as below
+When that version is not installed
+Remove will fail
+Grep for the specific error message
+
+sudo /etc/tedge/sm-plugins/apt remove rolldice  --module-version 1.111111
+
+"""
+
+
+class AptPluginRemoveWithVersionFails(AptPlugin):
+    def setup(self):
+        super().setup()
+        self.version = "1.111111"
+        self.package = "rolldice"
+        self.plugin_cmd("install", "outp_install", 0, argument=self.package)
+        self.addCleanupFunction(self.cleanup_remove_rolldice_module)
+
+    def execute(self):
+        self.plugin_cmd(
+            "remove", "outp_remove", 5, argument=self.package, version=self.version
+        )
+       
+
+    def validate(self):
+       self.assertGrep("outp_remove.err", "ERROR: Package with the specified `1.111111` version not installed")
+
```

Comment:
It's better to do an additional validation here that the existing package that was installed in the setup phase is still present and not accidentally removed.

Reply:
done